### DISCUSSION TITLE
[FEATURE REQ] Extended rTorrent functionality for ratio groups and priority

### DISCUSSION THREAD

**Discussion by A69Tor** [autobrr/autobrr#1667] [Ideas]
**Describe the solution you'd like**
Allow extended functionality when interacting with rTorrent as an action, such as rTorrent ratio groups and rTorrent priority

**Comment by dash82** [comment]
This would be very handy when dealing with different providers so you can set the ratio group and the retention period depending on the provider in your filter.

**Comment by EdTheLoon** [comment]
I agree with this request. I got bombed with hit and run pre-warnings as a result of one tracker having minimum seed time requirement regardless of ratio.

**Comment by eTheBlack** [comment]
Why this still wasnt implented? This is a must

**Comment by ghost** [comment]
> Why this still wasnt implented? This is a must

Hi, this is a hobby project for all of us, and we have a limited amount of time to work on priority fixes/features that we ourselves designate. If you consider this a must-have, then please submit a pull request. We'd gladly check it out.

**Reply by n1ce-0ne** [reply]
wish i could, i enjoy coding too, ive never looked at this code & ive never submitted a pull request before

**Comment by n1ce-0ne** [comment]
+1 for this, i'd love to be able to set ratio groups with different filters.

**Comment by sk1llsh0t** [comment]
I would also love to have ratio groups added to the actions for rTorrent
